Example/DApp/Modules/Sign/SessionAccount/SessionAccountPresenter.swift

```diff
@@ -41,14 +44,21 @@ final class SessionAccountPresenter: ObservableObject {
         do {
             let requestParams = try getRequest(for: method)
             
-            let request = Request(topic: session.topic, method: method, params: requestParams, chainId: Blockchain(sessionAccount.chain)!)
+            let ttl: TimeInterval = 300
+            let request = try Request(topic: session.topic, method: method, params: requestParams, chainId: Blockchain(sessionAccount.chain)!, ttl: ttl)
```

Comment:
ttl is configuring from client side now?

Reply:
yes, it's easier to init for the sdk consumer and we are counting under the hood